### ISSUE TITLE
Add option to disable color flash effects

### ISSUE THREAD

**Issue by user8595** [user8595/stak#5]
Since the entire functionality is handled in simple update functions, this should be easy to implement.

**Comment by user8595** [comment]
Might also consider adding a feature to disable the "after image" effects.

**Comment by user8595** [comment]
Somehow, i felt dizzy by just looking at the game over flash however.. should be urgent then

**Comment by user8595** [comment]
That was quick to add.
